### DISCUSSION TITLE
Missing "Remove All Events" button in mirth 4.2.0

### DISCUSSION THREAD

**Discussion by IbrahimAlhaj9** [nextgenhealthcare/connect#5603] [General]
Hi Everyone , 
When I upgraded the mirth version from 4.1.1 to 4.2.0 , we lost the "Remove All Events" button, can you please assist why?

Thank you

**Comment by pacmano1** [comment]
It was removed for compliance reasons.  I remember seeing the thread either here or on Slack, just can't find it at the moment.

**Reply by ab-20-m** [reply]
Github Issue: [[SECURITY] Prevent Ability to Remove All Events #5424](https://github.com/nextgenhealthcare/connect/issues/5424)
Release Notes Note: [Removed the Remove All Events Button](https://github.com/nextgenhealthcare/connect/wiki/4.2.0---What's-New#removed-the-remove-all-events-button)

**Reply by IbrahimAlhaj9** [reply]
Thank you

**Comment by siddharthsharma9258** [comment]
The underlying reason is Interesting.
